src/john.c

```diff
@@ -1780,8 +1778,14 @@ static void john_run(void)
 
 		omp_autotune_run(&database);
 
-		if (trigger_reset)
```

Comment:
I'm pretty sure this `trigger_reset` thing are from back in the times where `--stdout`'s `dummy_fmt->methods.reset` was NULL so would segfault.